client/app.jsx

```diff
@@ -16,32 +17,64 @@ export default class App extends React.Component {
       route: parseRoute(window.location.hash),
       messages: []
     };
+    this.socket = socket;
```

Comment:
Since it is imported at the top of the file you don't need to still assign it to a property on the class, not hurting anything but also not necessary

Reply:
I think I can do that for the functional components but for this one in particular removing that line breaks the page.